crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -153,12 +153,18 @@ impl<Client: HeaderProvider + BlockProvider + StateProvider> EngineApi<Client> {
              return Ok(PayloadStatus::from_status(PayloadStatusEnum::Syncing))
         };
 
-        if let Some(parent_td) = self.client.header_td(&parent_hash)? {
-            if Some(parent_td) <= self.chain_spec.paris_status().terminal_total_difficulty() {
-                return Ok(PayloadStatus::from_status(PayloadStatusEnum::Invalid {
-                    validation_error: EngineApiError::PayloadPreMerge.to_string(),
-                }))
-            }
+        let parent_td = if let Some(parent_td) = self.client.header_td(&block.parent_hash)? {
+            parent_td
+        } else {
```

Comment:
think this may work too

```suggestion
        let Some(parent_td) = self.client.header_td(&block.parent_hash)? else {
```